source/includes/entityref/_employeepay.md

```diff
@@ -0,0 +1,141 @@
+# EmployeePay
+
+Entity that represents a candidate's paycheck and payroll information.
+
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>EmployeePay fields</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="even">
+            <td>employeePayID</td>
+            <td>Integer</td>
+            <td>The Payroll Provider’s identifier for the employee’s payment. This would be separate from Check ID or Check Number. This is required to update the Paycheck UI for your client.</td>
```

Comment:
Spell out user interface rather than user UI throughout

Reply:
UI has been corrected to user interface.